### PR TITLE
Add support for Django client driver

### PR DESCRIPTION
**Pull request by michelts**
The splinter API is very useful to traverse over a Django client response body, but there is a minor bug that prevents one to use django as driver with pytest-splinter.

I fixed the bug and added a test for it.

Let me know if I missed anything!

### CODE REVIEW

File: pytest_splinter/plugin.py [hunk 1]
```diff
@@ -70,9 +70,9 @@ def Browser(*args, **kwargs):
     visit_condition = kwargs.pop('visit_condition')
     visit_condition_timeout = kwargs.pop('visit_condition_timeout')
     browser = splinter.Browser(*args, **kwargs)
-    browser.switch_to = browser.driver.switch_to
     browser.wait_for_condition = functools.partial(_wait_for_condition, browser)
     if hasattr(browser, 'driver'):
+        browser.switch_to = browser.driver.switch_to
```
Comment (bubenkoff) [inline pytest_splinter/plugin.py:75]: implement a simple test which tests only this change, instead of emulating the whole django app

Comment (michelts) [reply pytest_splinter/plugin.py:75]: Hi @bubenkoff 

You are right, sorry about that! I dropped the Django sample project and mocked the splinter driver to raise AttributeError when hitting browser.switch_to.

**Comment by michelts** [comment]
Done! Thank you @bubenkoff

**Comment by bubenkoff** [comment]
@michelts added you to the repo, please merge by squashing the commits into one

**Comment by bubenkoff** [comment]
@michelts released 2.1.0

**Pull request merged**